packages/translation/src/lang/en.ts

```diff
@@ -1070,6 +1070,41 @@ export default {
         internalServerError: "Failed to fetch indexers status",
       },
     },
+    healthMonitoring: {
+      name: "System Health Monitoring",
+      description: "Displays information showing the health and status of your system(s).",
+      option: {
+        fahrenheit: {
+          label: "CPU Temp in Fahrenheit",
+        },
+        cpu: {
+          label: "Show CPU Info",
+        },
+        memory: {
+          label: "Show Memory Info",
+        },
+        fileSystem: {
+          label: "Show Filesystem Info",
+        },
+      },
+      popover: {
+        processor: "Processor:",
+        version: "Version:",
+        uptime: "Uptime: {days} days, {hours} hours",
+        loadAverage: "Load Average",
+        oneMinute: "1 minute:",
+        fiveMinutes: "5 minutes:",
+        fifteenMinutes: "15 minutes:",
+        total: "Total:",
+        used: "Used",
+        diskAvailable: "Available",
+        memAvailable: "Available:",
+      },
+      memory: {},
```

Comment:
Is this one used or can it be removed?

Reply:
I use colons. If I add the colons in the code, then in left-to-right languages ​​they will remain from left to right.